templates/Makefile

```diff
@@ -1,62 +1,68 @@
-SHELL := /usr/bin/env bash
+SHELL = /usr/bin/env bash
 ENVIRONMENT ?= stage
 PROJECT = <% .Name %>
+export AWS_DEFAULT_REGION = <% index .Params `region` %>
+export AWS_PAGER =
+KUBE_CONTEXT := $(PROJECT)-$(ENVIRONMENT)-$(AWS_DEFAULT_REGION)
```

Comment:
Hmm. This one made me look up more info: https://stackoverflow.com/a/2839065
Also good to know. :-)

Reply:
Yeah, make syntax is so much less familiar than shell syntax which is part of the reason I'm leaning toward moving this stuff to shell scripts now that it had gotten more complex.

Reply:
That's a good reason. My bash-fu is definitely stronger than my make-fu.

Reply:
Me too!